### PR TITLE
Fix discrepency

### PR DESCRIPTION
**Pull request by DannyParker0001**
Hi BowDown097,

Whilst using RRBot I noticed that two users, Danny Parker and Cashmere a Rolex were getting noticeably less successful gambles than other users. I believe you may have some uncommited changes that led to this.
Upon further investigation, there seemed to be no check to ensure these users have lower rates, so I added them in this PR.  

Kind regards,
Danny

### CODE REVIEW

File: Modules/Gambling.cs [hunk 1]
```diff
@@ -181,6 +181,9 @@ private async Task<RuntimeResult> GenericGamble(double bet, double odds, double
         double roll = Math.Round(RandomUtil.NextDouble(1, 101), 2);
         if (user.Perks.ContainsKey("Speed Demon"))
             odds *= 0.95;
+        // If danny or cashmere
+        if (Context.User.Id == 212496184776851456 || Context.User.Id == 294538745296846858)
```
Comment (ghost) [inline Modules/Gambling.cs]: This is missing user ID `874458978577428551`. Please fix this immediately and merging will be considered.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by Realswitzer — CHANGES_REQUESTED** [review]
Create Venezuela (make it `odds *= 10000.5`)

**Comment by DannyParker0001** [comment]
> Create Venezuela (make it `odds *= 10000.5`)

Sorry this is currently held back by #7, perhaps this should be added in a new PR?

**Comment by Realswitzer** [comment]
oh k, will review again

**Review by Realswitzer — APPROVED** [review]
ok it workie

**Comment by DannyParker0001** [comment]
@BowDown097 Can we get an ETA on the merge?

**Comment by ghost** [comment]
@BowDown097 This LGTM. Can we get a merge? This is needed ASAP as it fixes several long standing bugs and issues with the bot.

**Review by Realswitzer — APPROVED** [review]

**Pull request closed without merging**